src/widgets/appointments/appointment-record.component.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react";
+import SummaryCard from "../../ui-components/cards/summary-card.component";
+import AppointmentsForm from "./appointments-form.component";
+import { openMedicationWorkspaceTab } from "../medications/medication-orders-utils";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import dayjs from "dayjs";
+import { getAppointments } from "./appointments.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { isEmpty } from "lodash-es";
+import { useRouteMatch } from "react-router-dom";
+import VerticalLabelValue from "../../ui-components/cards/vertical-label-value.component";
+import styles from "./appointment-record.css";
+
+export default function AppointmentRecord(props: AppointmentRecordProps) {
+  const [patientAppointment, setpatientAppointment] = useState(null);
+  const match = useRouteMatch();
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  const [startDate, setStartDate] = useState(dayjs().format());
+
+  useEffect(() => {
+    if (patientUuid && match.params) {
+      const abortController = new AbortController();
+      getAppointments(patientUuid, startDate, abortController).then(
```

Comment:
Perhaps it would be worth figuring out a way to fetch an Appointment via uuid if the API supports it?